src/Directory.build.props

```diff
@@ -1,20 +1,16 @@
 <Project>
     <!-- Project properties -->
     <PropertyGroup>
-        <TargetFrameworks>net45;net462</TargetFrameworks>
-        <ExtrasEnableWpfProjectSetup>true</ExtrasEnableWpfProjectSetup>
-        <ExtrasDefaultResourceIncludes>Resources\*.xaml</ExtrasDefaultResourceIncludes>
         <OutputPath>$(MSBuildThisFileDirectory)\bin\$(Configuration)\$(MSBuildProjectName)</OutputPath>
         <AppendTargetFrameworkToOutputPath>true</AppendTargetFrameworkToOutputPath>
         <GenerateDocumentationFile>false</GenerateDocumentationFile>
         <AutoGenerateBindingRedirects>true</AutoGenerateBindingRedirects>
+		<DisableImplicitFrameworkReferences Condition="'$(TargetFramework)' != 'netcoreapp3.0'">true</DisableImplicitFrameworkReferences>
```

Comment:
Why do we need this?

Reply:
This removes the standard Framework references, like System.Xml, System.Xml.Linq and gives you control over what Framework references are used. It reduces the clutter in the project, maybe even reduce build time, and makes the nupkg a bit cleaner.

On the other side, you need to specify the used FrameworkReferences manually.

It depends a bit on how much control one wants... doesn't really hurt to remove it, than you can remove the Reference entries for ! netcoreapp3.0 further ahead.